### ISSUE TITLE
Action bar greeting/farewell

### ISSUE THREAD

**Issue by oskarkk** [EngineHub/WorldGuard#1658]
**Is your feature request related to a problem? Please describe.**

There are flags for greeting/farewell that appears in the chat or as a title. Both types of flags have their disadvantages: the chat one spams the chat if user is entering/leaving regions many times, and the title one is too big and prominent for small informative greetings.

These are the flags I'm talking about:

> greeting | string | The message that appears in chat upon entering the region
> greeting-title | string | The title that appears upon entering the region. Including a newline (\n) will send a subtitle.
> farewell | string | The message that appears in chat upon leaving the region
> farewell-title | string | The title that appears upon leaving the region. Including a newline (\n) will send a subtitle.

**Describe the solution you'd like**

I propose adding two new flags, maybe named "greeting-bar" and "farewell-bar", that would show an action bar when entering/leaving the region. It'd work like the command:

` /title @s actionbar "test greeting"`

**Describe alternatives you've considered**

I believe there aren't any plugins with flags like that. It could be done with console-command-on-entry/console-command-on-exit in WorldGuard Extra Flags, but that seems like an overcomplicated solution for a simple feature like that.

**Comment by wizjany** [comment]
I considered it when I wrote the title flag but the action bar stuff was WIP in text at the time. Might end up waiting for the switch to adventure since the API is way better now.

**Comment by DevJoey** [comment]
This feature would be nice but it would also be nice if there was a flag to prevent farewell title/actionbar to be send if i teleport out of the region.

**Comment by accZerek** [comment]
Any update on the possibility of action bar greetings/farewells?

For those who stumble here via google search etc, you can accomplish this for the time being [withworldguard extra flags](https://www.spigotmc.org/resources/worldguard-extra-flags.4823/), running a vanilla title-action bar command with `console-command-on-entry` flag. 

Note that WorldGuard does not troubleshoot worldguard extraflags

**Comment by TheRealArchie** [comment]
Yep I'd love this! Bump! If that's a thing you do on github anyway.

**Comment by oskarkk** [comment]
> This feature would be nice but it would also be nice if there was a flag to prevent farewell title/actionbar to be send if i teleport out of the region.

@DevJoey That's a nice suggestion. Since I've made this issue I've been using `console-command-on-entry`/`console-command-on-exit solution` (which, by the way, has no uppercase characters because of a bug: aromaa/WorldGuardExtraFlagsPlugin/issues/164 - flagged as "wg bug" so maybe I should make an issue about it here?), and in many cases I wished that the farewell wouldn't appear on teleporting out of the region. 

There could be another flag like `greeting-on-teleport` which could be either boolean, or maybe better if it would be a string that would be the message that would appear only when teleporting into the region. If you wanted to have message on entry, you'd set `greeting` to something, and `greeting-on-teleport` to empty string or space. But then, what about other kinds of messages - title and actionbar? Making separate `greeting-on-teleport`, `greeting-title-on-teleport` and `greeting-bar-on-teleport`, and the same for farewell, maybe would be too much.
